pkg/apis/jws/v1alpha1/tomcat_types.go

```diff
@@ -26,6 +26,7 @@ type TomcatSpec struct {
 	ImageStreamNamespace string `json:"imageStreamNamespace"`
 	MavenMirrorUrl       string `json:"mavenMirrorUrl"`
 	ArtifactDir          string `json:"artifactDir"`
+	Size		     int32  `json:"size"`
```

Comment:
fix for  pkg/controller/tomcat/tomcat_controller.go:205:21: tomcat.Spec.Size undefined (type v1alpha1.TomcatSpec has no field or method Size)